2021-05-20 08:52:58,655 INFO    MainThread:43078 [wandb_setup.py:_flush():69] setting env: {}
2021-05-20 08:52:58,655 INFO    MainThread:43078 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-20 08:52:58,655 INFO    MainThread:43078 [wandb_init.py:_log_setup():336] Logging user logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/Classification/Heart-Disease-UCI/wandb/run-20210520_085258-ia4pfmrb/logs/debug.log
2021-05-20 08:52:58,655 INFO    MainThread:43078 [wandb_init.py:_log_setup():337] Logging internal logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/Classification/Heart-Disease-UCI/wandb/run-20210520_085258-ia4pfmrb/logs/debug-internal.log
2021-05-20 08:52:58,656 INFO    MainThread:43078 [wandb_init.py:_jupyter_setup():287] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f09b6a92438>
2021-05-20 08:52:58,656 INFO    MainThread:43078 [wandb_init.py:init():369] calling init triggers
2021-05-20 08:52:58,656 INFO    MainThread:43078 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-05-20 08:52:58,656 INFO    MainThread:43078 [wandb_init.py:init():418] starting backend
2021-05-20 08:52:58,656 INFO    MainThread:43078 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-20 08:52:58,659 INFO    MainThread:43078 [backend.py:ensure_launched():132] starting backend process...
2021-05-20 08:52:58,680 INFO    MainThread:43078 [backend.py:ensure_launched():137] started backend process with pid: 67639
2021-05-20 08:52:58,681 INFO    MainThread:43078 [wandb_init.py:init():423] backend started and connected
2021-05-20 08:52:58,692 INFO    MainThread:43078 [wandb_init.py:init():465] updated telemetry
2021-05-20 08:52:58,693 INFO    MainThread:43078 [wandb_init.py:init():484] communicating current version
2021-05-20 08:52:59,921 INFO    MainThread:43078 [wandb_init.py:init():489] got version response 
2021-05-20 08:52:59,922 INFO    MainThread:43078 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-20 08:53:00,509 INFO    MainThread:43078 [wandb_init.py:init():522] starting run threads in backend
2021-05-20 08:53:05,515 INFO    MainThread:43078 [wandb_run.py:_console_start():1565] atexit reg
2021-05-20 08:53:05,515 INFO    MainThread:43078 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.WRAP
2021-05-20 08:53:05,515 INFO    MainThread:43078 [wandb_run.py:_redirect():1455] Wrapping output streams.
2021-05-20 08:53:05,515 INFO    MainThread:43078 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-20 08:53:05,516 INFO    MainThread:43078 [wandb_init.py:init():546] run started, returning control to user process
2021-05-20 08:53:05,562 INFO    MainThread:43078 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-20 08:53:05,562 INFO    MainThread:43078 [jupyter.py:save_ipynb():268] looking for notebook: 00-main.ipynb
2021-05-20 08:53:06,268 INFO    MainThread:43078 [wandb_init.py:_pause_backend():253] saved code: <wandb.sdk.wandb_artifacts.Artifact object at 0x7f09b64bef60>
2021-05-20 08:53:59,824 INFO    MainThread:43078 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-20 08:53:59,827 INFO    MainThread:43078 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-20 08:53:59,828 INFO    MainThread:43078 [jupyter.py:save_ipynb():268] looking for notebook: 00-main.ipynb
2021-05-20 08:54:00,516 INFO    MainThread:43078 [wandb_init.py:_pause_backend():253] saved code: <wandb.sdk.wandb_artifacts.Artifact object at 0x7f09b6a9e5f8>
2021-05-20 08:54:00,519 INFO    MainThread:43078 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-20 08:54:00,519 INFO    MainThread:43078 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-20 08:54:00,520 INFO    MainThread:43078 [jupyter.py:save_ipynb():268] looking for notebook: 00-main.ipynb
2021-05-20 08:54:01,230 INFO    MainThread:43078 [wandb_init.py:_pause_backend():253] saved code: <wandb.sdk.wandb_artifacts.Artifact object at 0x7f09b8a6a160>
2021-05-20 08:54:01,234 INFO    MainThread:43078 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-20 08:54:01,234 INFO    MainThread:43078 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-20 08:54:01,234 INFO    MainThread:43078 [jupyter.py:save_ipynb():268] looking for notebook: 00-main.ipynb
2021-05-20 08:54:01,931 INFO    MainThread:43078 [wandb_init.py:_pause_backend():253] saved code: <wandb.sdk.wandb_artifacts.Artifact object at 0x7f09b6345240>
2021-05-20 08:54:01,934 INFO    MainThread:43078 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-20 08:54:01,934 INFO    MainThread:43078 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-20 08:54:01,934 INFO    MainThread:43078 [jupyter.py:save_ipynb():268] looking for notebook: 00-main.ipynb
2021-05-20 08:54:02,637 INFO    MainThread:43078 [wandb_init.py:_pause_backend():253] saved code: <wandb.sdk.wandb_artifacts.Artifact object at 0x7f09b66a2c88>
2021-05-20 08:54:02,644 INFO    MainThread:43078 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-20 08:54:02,662 INFO    MainThread:43078 [wandb_setup.py:_flush():69] setting env: {}
2021-05-20 08:54:02,662 INFO    MainThread:43078 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-20 08:54:02,662 INFO    MainThread:43078 [wandb_init.py:_log_setup():336] Logging user logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/Classification/Heart-Disease-UCI/wandb/run-20210520_085402-lk4deak6/logs/debug.log
2021-05-20 08:54:02,662 INFO    MainThread:43078 [wandb_init.py:_log_setup():337] Logging internal logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/Classification/Heart-Disease-UCI/wandb/run-20210520_085402-lk4deak6/logs/debug-internal.log
2021-05-20 08:54:02,663 INFO    MainThread:43078 [wandb_init.py:init():369] calling init triggers
2021-05-20 08:54:02,663 INFO    MainThread:43078 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-05-20 08:54:02,663 INFO    MainThread:43078 [wandb_init.py:init():391] re-initializing run, found existing run on stack: ia4pfmrb
2021-05-20 08:54:02,663 INFO    MainThread:43078 [wandb_run.py:finish():1208] finishing run ranuga-d/Heart-Disease-UCI/ia4pfmrb
